ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-cpou-loaded-upgrade-from-4.12.yaml

```diff
@@ -28,59 +28,22 @@ resources:
       memory: 200Mi
 tests:
 - always_run: false
-  as: loaded-upgrade-412to414-3nodes
-  steps:
-    allow_skip_on_success: true
-    cluster_profile: aws-perfscale-qe
-    env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
-      CHURN: "false"
-      CONTROL_PLANE_INSTANCE_TYPE: m5.xlarge
-      GC: "false"
-      ITERATION_MULTIPLIER_ENV: "6"
-      ZONES_COUNT: "3"
-    post:
-    - chain: ipi-aws-post
-    pre:
-    - chain: ipi-conf-aws
-    - ref: ovn-conf
-    - chain: ipi-install
-    test:
-    - as: set-upgrade-releases
-      commands: echo "${RELEASE_IMAGE_INTERMEDIATE},${RELEASE_IMAGE_TARGET}" > ${SHARED_DIR}/upgrade-edge
-      dependencies:
-      - env: RELEASE_IMAGE_INTERMEDIATE
-        name: release:intermediate
-      - env: RELEASE_IMAGE_TARGET
-        name: release:target
-      from: cli
-      resources:
-        requests:
-          cpu: 100m
-          memory: 200Mi
-    - chain: openshift-qe-upgrade-max-unavailable
-    - chain: openshift-qe-cluster-density-v2
-    - chain: openshift-upgrade-qe-sanity-cpou
-  timeout: 8h0m0s
-- always_run: false
-  as: loaded-upgrade-ipsec-412to414-3nodes
+  as: loaded-upgrade-412to414-24nodes
   steps:
     allow_skip_on_success: true
     cluster_profile: aws-perfscale-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       CHURN: "false"
+      COMPUTE_NODE_REPLICAS: "24"
       CONTROL_PLANE_INSTANCE_TYPE: m5.xlarge
       GC: "false"
       ITERATION_MULTIPLIER_ENV: "6"
+      MAX_UNAVAILABLE_WORKER: "3"
+      MCO_CONF_DAY2_CUSTOM_MCP: '[{"mcp_name": "infra"}]'
+      PAUSED_MCP_NAME: worker
```

Comment:
I see only worker nodes paused, it seems not a control-plane-only update, right?

Reply:
@jiajliu Yes, the job runs control-plane-only update.

Reply:
so both worker and infra mcp should be paused?

Reply:
@jiajliu I only pause worker mcp with `PAUSED_MCP_NAME: worker`. 
For `MCO_CONF_DAY2_CUSTOM_MCP: '[{"mcp_name": "infra"}]'`, this is to overwrite the `expected_mcp` that only allow master and worker mcp by default [here](https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/cpou/pause-worker-mcp/cucushift-upgrade-cpou-pause-worker-mcp-commands.sh#L45-L47)

Reply:
@jiajliu I only paused worker mcp `PAUSED_MCP_NAME: worker`
For `MCO_CONF_DAY2_CUSTOM_MCP: '[{"mcp_name": "infra"}]'`, it is to overwrite `expected_mcp` that only allow master and worker mcp by default https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/upgrade/cpou/pause-worker-mcp/cucushift-upgrade-cpou-pause-worker-mcp-commands.sh#L45-L47. This step 'check all actual mcp, if any of them unknown then break the job.'

Reply:
> I only paused worker mcp PAUSED_MCP_NAME: worker

Hmm, for a control-plane-only update, all non-master mcp should be paused before upgrade. If your test requirement is to do cpou update, I guess both infra and worker are expected to be paused.

Reply:
@jiajliu From perfscale test team's point of view, either is ok for us. It will be good if I can get some guidance about what is recommended officially.
The current way I did (infra mcp not paused) is based on a pr [21175](https://github.com/openshift/release/pull/21175), and related Jira [OTA-448:](https://issues.redhat.com/browse/OTA-448) Add upgrade tests for a cluster with infra nodes. From the description of pr https://github.com/openshift/release/pull/21175, my understanding is the infra mcp is expected to be unpaused. 
I will start a thread between the pr owner and us to clarify it.

Reply:
https://redhat-internal.slack.com/archives/CH76YSYSC/p1733456153871259